webviz_subsurface/plugins/_group_tree/group_tree.py

```diff
@@ -0,0 +1,215 @@
+from typing import List, Dict, Tuple, Callable
+
+import pandas as pd
+import dash
+import dash_html_components as html
```

Comment:
We should use the new `from dash import html` 🙂 https://dash.plotly.com/dash-2-0-migration

Reply:
to be explicit:
```suggestion
from dash import Dash, html
```
and further down replace `app: dash.Dash` with `app: Dash` to be consistent with the changes we made in other plugins for `Dash 2.0` 😊
The same import update needs to be made several places

Reply:
OK